docs/pages/hdmi_cec_source_tests.md

```diff
@@ -0,0 +1,176 @@
+| 1| [Physical Address](#physical-address)| Verify the valid physical address allocated through the HAL function.|HdmiCecGetPhysicalAddress| Y  |NA||
+| | | Verify the physical addresses allocated by connecting a source and sink device through an HDMI switch.|NA | NA | Y  | Enable the television connected to my Tele first so that it can declare its physical address first.  |
+
+#### Emulator Requirements - Physical Address
+
+- Boot control configuration to setup the cec network nodes
+  - A source and sink devices are connected to the network
+
+#### Control Plane Requirements - Physical Address
+
+- Control plane will allow removing or adding a node to the network.
+  - allowing adding/removing node sink node
+  - allowing adding/removing node source node
+
+### CEC Synchronous Transmission
+
+|S.No.|Test Functionality|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----|------------------|-----------|--------|--|--|--------------------------|
+| 3| [CEC Transmission](#cec-transmission)| Verify the correct transmission of the supported CEC commands (as per 1.4b HDMI CEC spec) to the connected device and ensure it is acknowledged properly.| HdmiCecTx | NA | Y  ||
+| | | Broadcast a supported CEC Command to all the devices connected to the network without any error |HdmiCecTx| NA | Y  ||
+| | | Transmit a CEC Command (as per 1.4b HDMI CEC spec) to put the connected device into standby mode and await the device's response. Monitor the behaviour of the connected device accordingly.| HdmiCecTx | NA | Y  | Control panel to monitor the behavior of the connected devices.  |
+| | | Transmit a CEC Command (as per 1.4b HDMI CEC spec) to get the CEC Version for a logical address that doesn't exist.| HdmiCecTx | Y | NA  |   |
+
+#### Emulator Requirements - CEC Transmission
+- Boot configuration
+  - Min case scenario multiple network nodes
+  - Max case scenario multiple network nodes
+
+#### Control Plane Requirements - CEC Transmission
+- The control plane will allow putting nodes into standby mode, this will cause a CEC message on the network
+
+### CEC Receive functionality
+
+|S.No.|Test Functionality|Description|L2|L3|Control plane requirements|
+|-----|------------------|-----------|--|--|--------------------------|
+| 4| [CEC Receive functionality](#cec-receive-functionality)| Transmit a CEC Command that expects a response (Eg. GetCECVersion) to a connected device and see the response is received correctly. | NA |Y| |
+| | | Transmit all the CEC Broadcast Command (as per 1.4b HDMI CEC spec) which expects a response from all the connected devices like `<GetPhysicalAddress>`  | NA | Y  ||
+| | | Transmit Different CEC commands from the connected devices and consider the Acknowledgement and responses are received correctly from the host device (my TV here)| NA | Y  | Control panel to control the third-party devices to Transmit the required commands to my Device (Device under test) |
```

Comment:
Remove the references to "my".  Also mention the command to be used as ChatGPT need to understand that info while generating the code.